src/agent/src/linux_abi.rs

```diff
@@ -10,7 +10,7 @@ pub const SYSFS_DIR: &str = "/sys";
 pub const SYSFS_PCI_BUS_PREFIX: &str = "/sys/bus/pci/devices";
 pub const SYSFS_PCI_BUS_RESCAN_FILE: &str = "/sys/bus/pci/rescan";
 #[cfg(any(
-    target_arch = "powerpc64le",
+    target_arch = "powerpc64",
```

Comment:
Considering we already changed the ARCH to be powerpc64le, can't we enforce it to also match powerpc64le here?
What I'm worried is about the not being consistent in our name and having it biting is in the future (as it already did now).

Reply:
The problem is, I couldn't find a way for rust core to  identify powerpc64le instead of powerpc64! I guess, it needs rust lang to be built with the changes which is probably out of scope for this PR.

Reply:
So if I understand correctly, the rust target architecture is generic for ppc - `powerpc64` whereas the compiler is specific for `ppc64le`.  Similarly there might be compiler for `ppc64`. Is this the case ?

Reply:
This is the list of compiler toolchain that is available for power architecture:
```
powerpc-unknown-linux-gnu
powerpc64-unknown-linux-gnu
powerpc64le-unknown-linux-gnu (installed)
```
The installed one was installed by default by the rustup installer.

This is the list of target architecture by rust::core:
```
    x86
    x86_64
    arm
    aarch64
    mips
    mips64
    powerpc
    powerpc64
    nvptx
    wasm32
```

Reply:
The Makefile tries to determine the compiler toolchain by determining the host architecture through `uname` command which gives `ppc64le` as output

Reply:
Ack!

Reply:
Got it.. Thanks @abdasgupta